lib/src/pusher/pusher_client.dart

```diff
@@ -21,25 +21,40 @@ class PusherClient extends StreamHandler {
       const MethodChannel('com.github.chinloyal/pusher_client');
   static const classId = 'PusherClient';
 
-  static PusherClient? _singleton;
+  // static PusherClient? _singleton;
   void Function(ConnectionStateChange?)? _onConnectionStateChange;
   void Function(ConnectionError?)? _onConnectionError;
   String? _socketId;
 
-  PusherClient._(
+  /* PusherClient._(
     String appKey,
     PusherOptions options, {
     bool enableLogging = true,
     bool autoConnect = true,
   });
 
+   plugin:
+    platforms:
+      android:
+        package: com.github.chinloyal.pusher_client
+        pluginClass: PusherClientPlugin
+      ios:
+        pluginClass: PusherClientPlugin
+
+
+
+  */
+
+  //String? appKey;
+  //PusherOptions? options;
+
   /// Creates a [PusherClient] -- returns the instance if it's already be called.
-  factory PusherClient(
+  /* PusherClient(
```

Comment:
Hi, this code has serious error bro,
you commented whole of PusherClient constractor func,
could you please send me its correct version if you have it?